.github/workflows/swizzle-test.yml

```
@@ -0,0 +1,58 @@
+name: Swizzle Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['12']
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
```

```suggestion
        uses: actions/setup-node@v2
```

`actions/setup-node` also has a newer stable release `v2`: https://github.com/actions/setup-node/releases/tag/v2.1.4